plugins/chunter-impl/src/components/Comments.svelte

```diff
@@ -0,0 +1,58 @@
+<!--
+// Copyright © 2020 Anticrm Platform Contributors.
+// 
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import { Message as MessageModel, Comment } from '@anticrm/chunter'
+  import { getClient } from '@anticrm/workbench'
+  import { onDestroy, afterUpdate, createEventDispatcher } from 'svelte'
+  import chunter from '../plugin'
+  import Message from './Message.svelte'
+
+  export let message: MessageModel
+
+  const client = getClient()
+  let unsubscribe = () => {}
+
+  let div: HTMLElement
+  const dispatch = createEventDispatcher()
+  afterUpdate(async () => {
+    dispatch('update')
+  })
+
+  let comments: Comment[] = []
+  $: {
+    unsubscribe()
+    unsubscribe = client.query(chunter.class.Comment, { replyOf: message._id }, (result) => {
+      comments = result
+    })
+  }
+
+  onDestroy(() => {
+    unsubscribe()
+  })
+</script>
+
+<div class="channel-container" bind:this={div}>
+  {#each comments as m (m._id)}
+    <Message message={m} />
```

Comment:
`thread={true}` seems to be necessary here

Reply:
Nice, resolved.